kover-gradle-plugin/src/functionalTest/kotlin/kotlinx/kover/gradle/plugin/test/functional/cases/VerificationTests.kt

```diff
@@ -97,14 +97,14 @@ internal class VerificationTests {
 
         run("koverHtmlReport", "koverVerify", errorExpected = true) {
             verification {
-                assertKoverResult("""Rule 'counts rule' violated:
-  lines covered percentage is 46.590900, but expected minimum is 58
-  lines covered count is 41, but expected maximum is 3
+                assertResult("""Rule 'counts rule' violated:
+  lines covered count is *, but expected maximum is 3
```

Comment:
`*` means "any number". 
Because the percentage of instruction coverage may vary slightly between compiler versions. This test does not check specific values, so it is allowed to skip the actual coverage value